### ISSUE TITLE
Inject WR Events directly into the Digitizer Block

### ISSUE THREAD

**Issue by alexxcons** [fair-acc/gr-digitizers#54]
In GitLab by @alexxcons on May 3, 2020, 22:33

Either via callback, or via some "WR Receiver" Block

Reasoning:
- Some blocks require WR-Metainformation
- Match against trigger tags already in the source (only once, instead of once per channel, like it was before)

Note:
Digitizer should also be able to work w/o FESA injecting events

Note:
"WR Receiver" Block seems to be slow, possibly simpler to directly use callback on Digitizer Block See: https://gitlab.com/al.schwinn/DigitizerClass2/-/issues/2#note_129667613

**Comment by alexxcons** [comment]
In GitLab by @alexxcons on May 3, 2020, 23:55

Duplicate of https://gitlab.com/al.schwinn/DigitizerClass2/-/issues/88

**Comment by alexxcons** [comment]
In GitLab by @alexxcons on May 3, 2020, 23:55

closed
